tests/test_TFUtil.py

```diff
@@ -385,6 +385,33 @@ def test_4D_Data_get_placeholder_flattened():
   assert res.shape[0] == 7 * 9 * 13
   assert len(res.shape) == 2
 
+
+def test_4D_nhwc_Data_get_placeholder_flattened():
+  import numpy as np
+  import numpy.testing as npt
```

Comment:
Don't do that. Why? You don't need that. Just write directly `numpy` and `numpy.testing`.